spark/src/test/scala/org/apache/spark/sql/delta/DescribeDeltaHistorySuite.scala

```diff
@@ -579,7 +579,7 @@ trait DescribeDeltaHistorySuiteBase
         val e = intercept[AnalysisException] {
           sql(s"describe history $table").show()
         }
-        Seq("DESCRIBE HISTORY", "only supported for Delta tables").foreach { msg =>
+        Seq("is not a Delta table").foreach { msg =>
           assert(e.getMessage.contains(msg))
         }
```

Comment:
```suggestion
        assert(e.getMessage.contains("is not a Delta table"))
```
(but why does it no longer mention the command name?)